unit/goto-programs/goto_trace_output.cpp

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************\
```

Comment:
Just realised - this still needs to be added to the Makefile as `cbmc` is using Make and CMake in parallel.